com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+| **Detect Paper White** | Whether the paper white value is detected from the data communicated by the display device. We strongly suggest to provide a calibration screen to let the user set this value depending on their screen and viewing experience. |
+| **- Paper White**      | Paper white value (in nits) set when it is not automatically detected from the display. |
+
+#### Custom
+
+Currently not supported by HDR Output. It is possible to decide whether to fallback on Neutral or ACES for outputting to an HDR device.
+
+#### External
+
+Not supported. Mostly because every different HDR screen used to display the content would need a different LUT. It is possible to decide whether to fallback on Neutral or ACES for outputting to an HDR device.
+
+## Control HDR via script
+
+In addition to controlling the above described Volume Component parameters through the camera volume stack, it is also possible to query the [HDROutputSettings](https://docs.unity3d.com/ScriptReference/HDROutputSettings.html). This can be used for example to:
+
+- Query the display brightness limits ([min](https://docs.unity3d.com/ScriptReference/HDROutputSettings-minToneMapLuminance.htmlhttps://docs.unity3d.com/ScriptReference/HDROutputSettings-minToneMapLuminance.html) and [max](https://docs.unity3d.com/ScriptReference/HDROutputSettings-maxToneMapLuminance.html)) and [paper white value](https://docs.unity3d.com/ScriptReference/HDROutputSettings-paperWhiteNits.html).
+- [Request a change in HDR](https://docs.unity3d.com/ScriptReference/HDROutputSettings.RequestHDRModeChange.html), turning it on or off.
+
+## HDR Debug Views
+
+HDRP offers three debug views for HDR rendering. These can be found in **Window > Analysis > Render Pipeline Debugger > Lighting > HDR**
+
+#### Gamut View
+
+Will display on the bottom left of the screen two triangles representing the Rec709 and Rec2020 color gamuts and display what parts of the gamut are covered by the scene.
+![HDR-Output-GamutView](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-GamutView.png)
+
+This can be very useful to verify that you are taking advantage of the wider color gamut and is a way to see how color plot changes as you perform color grading.
+
+#### Gamut Clip
+
+Very similar to the above, except it shows red for areas of the screen that are outside the sRGB/Rec709 color gamut and green for anything that is in both the Rec709 and Rec2020 color gamut.
+
+![HDR-Output-GamutClip](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-GamutClip.png)
+
+
+
+#### Values over Paperwhite value
+
+This debug view shows the scene as luminance values except for parts of the scene that are over the paper white value that are displayed as a gradient from yellow (paperwhite +1) value to red (max brightness nits)
```

Comment:
Proposal:
"This debug view displays the Scene's luminance values. A gradient from yellow (**Paper White** +1) value to red (**Maximum Nits**) indicates values that exceed the **Paper White** value."

Reply:
Revised proposal:
"This debug view applies a colored gradient to the Scene to indicate regions in which the Scene's luminance exceeds the Paper White value. The gradient ranges from yellow, which represents the Paper White value +1, to red, which represents the Maximum Nits value."

Reply:
@FrancescoC-unity